lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac.rb

```diff
@@ -0,0 +1,33 @@
+module MiqAeMethodService
+  module MiqAeServiceRbac
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def find_ar_object_by_id(id)
+        Rbac.search(:class => model, :targets => [id],
+                    :results_format => :objects).first.first
+      end
+
+      def all
+        wrap_results(Rbac.search(:class => model, :results_format => :objects).first)
```

Comment:
@kbrock 
The Automate Engine code has been changed to set the User.current_user on entry, so in the scope of Automate the User.current_user is going to be set.